src/store/utils/settings.ts

```diff
@@ -31,6 +31,12 @@ export const wipeApp = async ({
 	restartApp?: boolean;
 } = {}): Promise<Result<string>> => {
 	try {
+		// stop onchain wallet if it exists
+		try {
+			const wallet = getOnChainWallet();
+			await wallet.stop();
+		} catch (e) {}
+
```

Comment:
We can expect the wallet to exist at this point, so it should be able to be stopped. If it does fail we should probably propagate and return the error. Especially if we're going to remove the restart in the future.

```suggestion
		// stop onchain wallet if it exists
		const wallet = getOnChainWallet();
		await wallet.stop();

```

Reply:
We run wipeApp during Onboard/TermsOfUse, at this point wallet does not exists

Reply:
Right ok let's leave it for now. If we're going to remove the restart later we should probably return the error outside the onboarding though.